src/main/java/Anke.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Scanner;
+
+public class Anke {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm Anke.");
+        System.out.println("What can I do for you?\n");
+        String line = "";
+        Task[] tasks = new Task[100];
+        int count = 0;
+        while (!line.equals("bye")) {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            Task task = null;
+            if (line.equals("bye")) {
+                break;
+            } else if (line.equals("list")) {
+                for (int i = 0; i < count; i++) {
+                    System.out.println((i + 1) + ". " + tasks[i].toString());
+                }
+                System.out.println("");
+                continue;
+            } else if (line.length() > 5 && line.substring(0, 5).equals("mark ")) {
+                int index;
```

Comment:
Could you declare the "index" variable within the try block, in the same line as the initialisation (line 25)?